client/src/components/footer/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import {
+  FooterBlock,
+  FooterLogo,
+  FooterLogoTitle,
+  FooterLogoSubTitle,
+  FooterDescription,
+  FooterList,
+  FooterContactTitle,
+  FooterContactInfo,
+} from "./footer";
+import { FlexContainerSpace, FlexStartColumn } from "../../box/flexBox";
+
+const Footer = () => {
+  const contacts = [
+    { id: 1, title: "Address:", info: "481 Creekside Lane Avila CA" },
+    { id: 2, title: "Phone:", info: "+53 345 7953 32453" },
+    { id: 3, title: "Email:", info: "yourmail@gmail" },
+  ];
```

Comment:
you can move this constant above component initialisation. we don't need to initialise this constant on each render.

Reply:
agree with you